.spelunky2/Locations.py

```diff
@@ -38,38 +38,38 @@ class Spelunky2LocationData(NamedTuple):
+    "LISE Project Journal Entry": Spelunky2LocationData(24, "Volcana"), # Reduced spawn rate until world 4 is entered
+    "Coco Von Diamonds Journal Entry": Spelunky2LocationData(25, "Vlad's Castle"),
     "Manfred Tunnel Journal Entry": Spelunky2LocationData(26, "Olmec's Lair"),
-    "Little Jay Journal Entry": Spelunky2LocationData(27, "Tide Pool"),
-    "Tina Flan Journal Entry": Spelunky2LocationData(28, "Tide Pool"),
+    "Little Jay Journal Entry": Spelunky2LocationData(27, "Tide Pool"), # Reduced spawn rate until a Tiamat win is completed
+    "Tina Flan Journal Entry": Spelunky2LocationData(28, "Abzu"), # Reduced spawn rate until a Tiamat win is completed
     "Valerie Crump Journal Entry": Spelunky2LocationData(29, "Temple"),
-    "Au Journal Entry": Spelunky2LocationData(30, "Temple"),
+    "Au Journal Entry": Spelunky2LocationData(30, "City of Gold"),
     "Demi Von Diamonds Journal Entry": Spelunky2LocationData(31, "Ice Caves"),
-    "Pilot Journal Entry": Spelunky2LocationData(32, "Ice Caves"),
-    "Princess Airyn Journal Entry": Spelunky2LocationData(33, "Neo Babylon"),
+    "Pilot Journal Entry": Spelunky2LocationData(32, "Mothership"),
+    "Princess Airyn Journal Entry": Spelunky2LocationData(33, "Neo Babylon"), # Reduced spawn rate until a Tiamat win is completed without shortcuts
     "Dirk Yamaoka Journal Entry": Spelunky2LocationData(34, "Sunken City"),
     "Guy Spelunky Journal Entry": Spelunky2LocationData(35, "Neo Babylon"),
     "Classic Guy Journal Entry": Spelunky2LocationData(36, "Sunken City"),
-    "Terra Tunnel Journal Entry": Spelunky2LocationData(37, "Menu"),  # Can be in either Jungle or Volcana
-    "Hired Hand Journal Entry": Spelunky2LocationData(38, "Dwelling"),
+    "Terra Tunnel Journal Entry": Spelunky2LocationData(37, "Any World 2"),  # Can be in either Jungle or Volcana
+    "Hired Hand Journal Entry": Spelunky2LocationData(38, "Dwelling"), # From Alto's coffin or a shop
     "Eggplant Child Journal Entry": Spelunky2LocationData(39, "Ice Caves"),
     "Shopkeeper Journal Entry": Spelunky2LocationData(40, "Dwelling"),
-    "Tun Journal Entry": Spelunky2LocationData(41, "Menu"),  # Can be in either Jungle or Volcana - Can be met in Dwelling by random chance
-    "Yang Journal Entry": Spelunky2LocationData(42, "Dwelling"),
-    "Madame Tusk Journal Entry": Spelunky2LocationData(43, "Tide Pool"),
-    "Tusk's Bodyguard Journal Entry": Spelunky2LocationData(44, "Tide Pool"),
-    "Waddler Journal Entry": Spelunky2LocationData(45, "Olmec's Lair"),
-    "Caveman Shopkeeper Journal Entry": Spelunky2LocationData(46, "Dwelling"),
-    "Ghist Shopkeeper Journal Entry": Spelunky2LocationData(47, "Tide Pool"),
+    "Tun Journal Entry": Spelunky2LocationData(41, "Any World 2"),  # Can be in either Jungle or Volcana - Can be met in Dwelling by random chance
+    "Yang Journal Entry": Spelunky2LocationData(42, "Dwelling"), # Can also be met in Black Market or Tide Pool 4-3

```

Comment:
Yang can only be met in TidePool if you didn't meet the sisters or Horsing. Otherwise they will appear instead.